README.md

```diff
@@ -15,6 +15,12 @@ production control planes.
 
 1. Install the [Upbound CLI][upbound-cli].
 
+   Homebrew:
+   ```console
+   brew install upbound/tap/up
+   ```
+   
+   Linux:
```

Comment:
nitpick: this isn't just for linux, and I think it should be our recommend strategy since it is the only one that works for all platforms

Reply:
Sounds good, though I think Homebrew instructions should exist in this section maybe as the second option.

What should we call it instead of `Linux`?

Reply:
Yeah I think if we just put the install script first with no label, and then follow with something like "To install with Homebrew:..."

Reply:
@hasheddan done!